### PR TITLE
add help links

### PR DESCRIPTION
**Pull request by andistuder**
adds help links at various places in the app.

### CODE REVIEW

File: app/views/feature_reviews/show.html.haml [hunk 1]
```diff
@@ -1,10 +1,13 @@
-- title "Feature Review"
+
 - if @feature_review_with_statuses.time
-  .lead.time= "Projecting events up to #{@feature_review_with_statuses.time}"
+  .alert.alert-warning.time= "Feature review as of #{@feature_review_with_statuses.time}"
```
Comment (dideler) [inline app/views/feature_reviews/show.html.haml]: This text seems a bit awkward (the old text did too). Would it sound better/be more clear if we said

> Viewing Feature Review as appeared at \<DateTime\>

File: app/views/releases/show.html.haml [hunk 2]
```diff
@@ -1,4 +1,12 @@
-- title "Releases for #{@app_name}"
+- title "Releases for #{@app_name}", help_url: 'https://github.com/FundingCircle/shipment_tracker/wiki/5.-MERGING-AND-DEPLOYING-A-FEATURE'
+
+%p
+  The Releases page on Shipment tracker can tell you what commits are pending deployment or have already been
+  deployed for your repository and whether they have been approved by a Product Owner. From this page you can see:
```
Comment (dideler) [inline app/views/releases/show.html.haml]: Could this be shortened to

> View which releases (merged commits) are pending deployment or have already been deployed, and whether any associated tickets have been approved in JIRA.

And drop the list below as it more or less repeats what was stated earlier.

Comment (chriswan84) [inline app/views/releases/show.html.haml]: Agreed. Lets change to what dennis said.

File: app/views/feature_reviews/show.html.haml (hunk 1 above)
Comment (chriswan84) [inline app/views/feature_reviews/show.html.haml]: The text for the flash notification should be:
"[clock glyph] As viewed at [Time]"

File: app/views/releases/show.html.haml (hunk 2 above)
Comment (chriswan84) [inline app/views/releases/show.html.haml]: Lets also move the page help links across to follow the page title.

**Pull request merged**